src/derivkit/calculus/hessian.py

```diff
@@ -20,111 +20,104 @@
 ]
 
 
-def build_hessian(function: Callable,
-                  theta0: np.ndarray,
-                  method: str | None = None,
-                  n_workers: int=1,
-                  return_diag: bool = False,
-                  **dk_kwargs: Any,
+def build_hessian(
+    function: Callable,
```

Comment:
```suggestion
    function: Callable[[ArrayLike], np.ndarray],
```
But note that `ArrayLike` should still be imported.